p2p/peer.py

```diff
@@ -956,10 +963,7 @@ def get_peers(self, min_td: int) -> List[BasePeer]:
                 for line in peer.get_extra_stats():
                     self.logger.debug("    %s", line)
             self.logger.debug("== End peer details == ")
-            try:
-                await self.sleep(self._report_interval)
-            except OperationCancelled:
-                break
```

Comment:
This was no longer necessary as this coroutine now runs via `BaseService.run_task()`